### ISSUE TITLE
Missing labels / accessible names for inputs and select elements across admin

### ISSUE THREAD

**Issue by aviannamiller** [omeka/Omeka#1069]
Currently, there are missing labels for the following:
- Appearance -> Navigation: `main_nav_checkboxes_#` inputs
- Settings -> Element Sets -> Edit Element Set: `elements-##-comment` textarea inputs

There are missing accessible names for the following:
- Item Type -> Add (or Edit) Item Type -> "Existing": `.existing-element-drop-down` `select` elements

**Comment by kimisgold** [comment]
The navigation inputs should have been labeled in 811436adf6dedb4e9e7a28e126e2a7d5676eafe5. Element set textarea inputs were addressed in 811436adf6dedb4e9e7a28e126e2a7d5676eafe5. 

I've provided labels for new and existing elements on the item type form on the [`item-type-labels`](https://github.com/omeka/Omeka/tree/item-type-labels) branch.

All of these are ready for testing.

**Comment by aviannamiller** [comment]
Everything is working well, however, there's some odd announcements for remove buttons on item type elements, seemingly coming from the labels. Remove buttons on "New" Elements announce as "Remove element name", if the input is not filled in. Buttons on "Existing" Elements (with nothing chosen on the select) announce as "Remove Select Below". If nothing is chosen or input on either, can the announcement be something like "Remove Unnamed Element" (...probably a better wording choice to be made here)?

**Comment by kimisgold** [comment]
I don't have a better idea than "Unnamed Element" for the moment, so I'm okay with that as the default for the time being. New elements will default to "Remove Unnamed Element" and then update to "Remove [Input Value]" if the text input or select are changed. If they are reset to blank, they will go back to "Remove Unnamed Element".

**Comment by aviannamiller** [comment]
Great, that works wonderfully. This should now be good to go.
